OrbitCore/LinuxEventTracer.cpp

```diff
@@ -21,69 +22,78 @@
 #include "OrbitModule.h"
 #include "OrbitProcess.h"
 #include "Params.h"
-#include "PrintVar.h"
 #include "SamplingProfiler.h"
 #include "TimerManager.h"
 
-//-----------------------------------------------------------------------------
-void LinuxEventTracer::Start() {
-  PRINT_FUNC;
-#if __linux__
-  *m_ExitRequested = false;
-
-  m_Thread = std::make_shared<std::thread>(&LinuxEventTracer::Run, m_Pid,
-                                           m_Frequency, m_ExitRequested);
-  m_Thread->detach();
-#endif
-}
-
-//-----------------------------------------------------------------------------
-void LinuxEventTracer::Stop() { *m_ExitRequested = true; }
-
-//-----------------------------------------------------------------------------
-void LinuxEventTracer::Run(
```

Comment:
We should consider refactoring this monster-method.

Reply:
Definitely, but not for now.